### ISSUE TITLE
Push don't works on mobile

### ISSUE THREAD

**Issue by hooddy** [signal-csharp/Signal-Windows#219]
I have not recieve background messages even when backgroudn for Signal is enabled. I've recive message only when I open app. Is it by design?

**Comment by Tourniquet88** [comment]
I actually do get background messages but usually it takes a lot longer to get them via background services. They are not "pushed" when you get it.

**Comment by golf1052** [comment]
Yes, we don't support real push notifications because that requires server side support. The background notifications are enabled by a [background task that runs every 15 minutes](https://github.com/signal-csharp/Signal-Windows/blob/2933a1e84d06d2fa2fd99d757957889c2ceff46a/Signal-Windows/App.xaml.cs#L200). A 15 minute interval is the fastest that background tasks can run](https://docs.microsoft.com/en-us/windows/uwp/launch-resume/support-your-app-with-background-tasks#background-tasks).

I still wouldn't be surprised if the background task is flaky and isn't actually running every 15 minutes. There's an option to export the UI log but not the background task log. Will need to add that.

@hooddy If you want you can send your Signal-Windows.bg.log to signal.windows@gmail.com. Currently access it on mobile you'll need to enable the [Device Portal](https://docs.microsoft.com/en-us/windows/uwp/debug-test-perf/device-portal-mobile) and then browse to `LocalAppData\2383BenediktRadtke.SignalPrivateMessenger_0.2.26.0_arm__teak1p7hcx9ga\LocalCache`. Use the save icon to save the `Signal-Windows.bg.log` file locally.

**Comment by hooddy** [comment]
@golf1052 i would like to send a log but version 3.0 just crashed on startup.

**Comment by golf1052** [comment]
You should be able to send logs without opening the app by using the Device Portal instructions.

**Comment by hooddy** [comment]
It says This folder has no data associated with it

**Comment by golf1052** [comment]
That's the folder it should be located in. Did you have the app installed before? And did it crash even before getting to the initial landing page?

**Comment by hooddy** [comment]
Previous one was installed. But this one wont install over it. I forced to remove it before install.

**Comment by golf1052** [comment]
Does the file directory show any data for the previous version? I assume not because you uninstalled it. Very strange that your getting a crash on startup with no data.

**Comment by hooddy** [comment]
I rollback to v0.2.26. Link with Android. Tryin to change some setting to save any of it in app dir. But there;s no signal folder on localcache. Send message to me from another account. Android phone receeve it. Windows phone doesn't.
![01](https://user-images.githubusercontent.com/6899176/108617688-6784fe80-7410-11eb-8f00-70de77634e20.PNG)
Btw i know why it can't instal over. v0.2.26 have trusted sign and v.0.3.0 doesn't.
![photo_2021-02-21_09-52-11](https://user-images.githubusercontent.com/6899176/108617853-5c7e9e00-7411-11eb-935b-6c26fa28132f.jpg)
![photo_2021-02-21_09-52-13](https://user-images.githubusercontent.com/6899176/108617855-5dafcb00-7411-11eb-9c19-c3bdf425190e.jpg)

**Comment by hooddy** [comment]
And theres log within 0.2.26 app
[1Signal-Windows.ui.log.txt](https://github.com/signal-csharp/Signal-Windows/files/6017338/1Signal-Windows.ui.log.txt)

**Comment by golf1052** [comment]
0.2.26 cannot receive messages anymore because of https://github.com/signal-csharp/Signal-Windows/issues/220

The problem may be that 0.3.0 isn't signed by the Microsoft Store yet because I haven't been able to deploy it there yet. I'm working with @Trolldemorted to get a new version to the store.

**Comment by Tourniquet88** [comment]
Yeah you can't upgrade an app with a different certificate. If you have the Github one installed before you should be able to upgrade just fine. That "untrusted message" because you probably did not install that certificate and you don't need to on mobile / 10x .

**Comment by hooddy** [comment]
@Tourniquet88 sure I know that bro. I don not get why in SM store not present Mobile device as requirements, it impossible to install it. When I open in PC browser its showm me Mobile device in requirements, but on my phone it doesn't and it in unavailable state and required a Reedem code. So it forced to use store.rg-adguard.net to grab it from MS Store.

**Comment by golf1052** [comment]
0.3.0 is in the store now. Hopefully that will fix the startup crashes.

**Comment by hooddy** [comment]
@golf1052 yep it runs. but linking don't works. it says "network error" or "no device found" on Android side.
